src/Shouldly.Tests/ShouldThrow/ActionDelegateScenario.cs

```diff
@@ -18,7 +18,7 @@ protected override string ChuckedAWobblyErrorMessage
         protected override void ShouldPass()
         {
             var ex = Should.Throw<NotImplementedException>(() => { throw new NotImplementedException(); });
-            ex.ShouldBeOfType<NotImplementedException>();
+            ex.ShouldBeOfType<NotImplementedException>(() => "Some additional context");
```

Comment:
`ActionDelegateScenario`, `FuncOfTaskOfStringScenario`, `FuncOfTaskOfStringThrowsDifferentExceptionScenario`, `FuncOfTaskScenario` and `FuncOfTaskScenarioAsync` - Fixed.